src/upload.test.ts

```diff
@@ -341,10 +351,58 @@ describe("splitSizeIntoChunkAlignedParts", () => {
         { start: 400 * mib, end: 500 * mib },
       ],
     ],
+
+    // Use larger chunk size.
+    [
+      40 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 40 * mib },
+      ],
+    ],
+    [
+      40 * mib,
+      3,
+      chunk * 3,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 0 },
+        { start: 0, end: 40 * mib },
+      ],
+    ],
+    [
+      41 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 41 * mib },
+      ],
+    ],
+    [
+      80 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 80 * mib },
+        { start: 80 * mib, end: 80 * mib },
+      ],
+    ],
+    [
+      81 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 80 * mib },
+        { start: 80 * mib, end: 81 * mib },
+      ],
+    ],
   ];
 
-  it.each(sizesAndChunks)("Should align size '%s' with '%s' parts", (totalSize, partCount, expectedParts) => {
-    const parts = splitSizeIntoChunkAlignedParts(totalSize, partCount);
+  it.each(sizesAndChunks)("('%s', '%s', '%s')", (totalSize, partCount, chunkSize, expectedParts) => {
```

Comment:
I think the text test name was nice, why did you remove it in favour of just listing the arguments?

Reply:
See discussion on https://github.com/SkynetLabs/skynet-js/pull/432#pullrequestreview-963222675.